apps/docs/src/app/platform/component-docs/platform-list/platform-list-examples/platform-list-with-single-selection-example.component.ts

```diff
@@ -32,11 +33,11 @@ export class ListDataProvider extends DataProvider<Address> {
 })
 export class PlatformListWithSingleSelectionExampleComponent {
     _dataSource = new ListDataSource<Address>(new ListDataProvider());
-    _selectedItems: any[] = [];
+    _selectedItem: string;
 
-    _showItemInfo(event: any): void {
+    _showItemInfo(event: SelectionChangeEvent): void {
         if (event.selectedItems[0] !== undefined) {
-            this._selectedItems = event.selectedItems[0].id;
+            this._selectedItem = event.selectedItems[0].id;
```

Comment:
Why are these with `_` ? Weren't we supposed to mark private properties and methods with `_`?

Reply:
Hi @InnaAtanasova , this property is used in component.html thats why we are marking these fields wiith _. if it is not used html and used only internal to component then private _XXX.